libs/host/Configuration/Options.cs

```diff
@@ -239,6 +239,10 @@ internal sealed class Options
         [Option("hcollect-freq", Required = false, HelpText = "Frequency in seconds for the background task to perform Hash collection. 0 = disabled. Hash collect is used to delete expired fields from hash without waiting for a write operation. Use the HCOLLECT API to collect on-demand.")]
         public int HashCollectFrequencySecs { get; set; }
 
+        [IntRangeValidation(0, int.MaxValue)]
```

Comment:
Can unify Z and H collection into one switch expired-object-collect-freq?

Reply:
Will do it

Reply:
This should be fine, its easy for clients to adapt to it.

Reply:
Done